.github/workflows/ci.yml

```diff
@@ -153,10 +153,49 @@ jobs:
           service_url: '${{ env.INTEGRATION_SERVICE_URL }}'
           requested_permissions: '{"scope":"integ","repositories":["github-token-minter"],"permissions":{"issues":"read"}}'
 
-      - name: 'verify'
+      - name: 'verify-github-token'
         run: |
           curl --fail \
             -H "Accept: application/vnd.github+json" \
-            -H "Authorization: Bearer ${{ steps.mint-token.outputs.token }}"\
+            -H "Authorization: Bearer ${{ steps.mint-github-token.outputs.token }}"\
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            https://api.github.com/repos/abcxyz/github-token-minter/issues/events
+
+      - id: 'auth'
+        name: 'Authenticate to Google Cloud'
```

Comment:
Can you document why this is needed, if someone comes to read this it might be confusing as to why we are doing the auth twice.

First token is to generate an id token that impersonates a service account to call the minty endpoint.

Second token is to generate an id token that impersonates a service account to  token requesting access to the minty repository.

Reply:
Added comments explaining the two separate tokens